esptool.py

```diff
@@ -18,6 +18,8 @@
 # this program; if not, write to the Free Software Foundation, Inc., 51 Franklin
 # Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+from __future__ import print_function, unicode_literals, division
+
```

Comment:
If future is going to be required, then it should be part of the flake8 configuration too, so it'll reject any new files without these future headers.